pkg/lib/localstore/objectstore/queries_test.go

```diff
@@ -251,14 +263,32 @@ func TestQuery(t *testing.T) {
 			bundle.RelationKeyLayout:      pbtypes.Int64(int64(model.ObjectType_relationOption)),
 		}
 
+		relObjDeleted := TestObject{
+			bundle.RelationKeyId:          pbtypes.String("relid2"),
+			bundle.RelationKeyRelationKey: pbtypes.String("bsonid1"),
+			bundle.RelationKeyName:        pbtypes.String("deletedtag"),
+			bundle.RelationKeyIsDeleted:   pbtypes.Bool(true),
+			bundle.RelationKeyDescription: pbtypes.String("this is a deleted relation's description"),
+			bundle.RelationKeyLayout:      pbtypes.Int64(int64(model.ObjectType_relationOption)),
+		}
+
+		relObjArchived := TestObject{
+			bundle.RelationKeyId:          pbtypes.String("relid3"),
+			bundle.RelationKeyRelationKey: pbtypes.String("bsonid1"),
+			bundle.RelationKeyName:        pbtypes.String("archived"),
+			bundle.RelationKeyIsDeleted:   pbtypes.Bool(true),
```

Comment:
@requilence this should be `bundle.RelationKeyIsArchived`?

Reply:
nope, it's just a name of the object I use to search :)